<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleDialog</class>
 <widget class="QWidget" name="SimpleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	
	background-color: rgba(0, 0, 0, 100);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>441</width>
       <height>185</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>441</width>
       <height>185</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget {
	border: 1px solid rgba(0,0,0,100);
border-radius: 4px;
	background-color: rgb(255, 255, 255);
}
QLabel{
		background-color: none;
}
QComboBox {
			background: rgba(0,0,0,10);
}

QSpinBox {
			background: rgba(0,0,0,10);
}

QDoubleSpinBox{
			background: rgba(0,0,0,10);
}

QLineEdit {
	background: rgba(0,0,0,10);
}

QPushButton {
		background-color: none;
	padding: 0 12px;
}

QPushButton::hover {
	background-color: rgba(0, 0, 0, 10);
	color: rgb(0, 0, 0);
}

QPushButton::checked {
	background-color: rgba(0, 0, 0, 20);
	color: rgb(0, 0, 0);
}

QPushButton::disabled {
	background-color: rgba(0, 0, 0, 20);
	color: rgb(0, 0, 0);
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QLabel" name="lblTitle">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblDetails">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::MarkdownText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnOk">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Abbrechen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SimpleDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
